face/table.go

```diff
@@ -80,3 +82,21 @@ func (t *Table) Remove(id uint64) {
 	table.Rib.CleanUpFace(id)
 	core.LogDebug("FaceTable", "Unregistered FaceID=", id)
 }
+
+// ExpirationHandler stops the faces that have expired
+func (t *Table) ExpirationHandler() {
+	for {
+		// Check for expired faces every 10 seconds
+		time.Sleep(10 * time.Second)
+
+		// Iterate the face table
+		t.faces.Range(func(_, face interface{}) bool {
```

Comment:
(Not related) Really want to see generics supported in `sync` package, but it seems to still have a long way to go: golang/go#47657

Reply:
Yeah, this is very annoying